lib/solidus/versions.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'version'
+
+module Solidus
+  class Versions
+    class << self
+      def from_yaml_data(data)
```

Comment:
How about having this as `Version.all` instead of having a separate `Versions` module?

Reply:
Unfortunately, data has to be passed from a view context so we need this `from_yaml_data` method, which is the same thing that we did to get the list of extensions from a .yml file.

For `Versions` vs `Version`, I opted for this path since `Version.from_yaml_data` makes me think that I'm instantiating a single version and not a collection. Anyway, I don't have strong opinions here and I can change that if you prefer this way.

Reply:
Got it. I think we could still move it to `Version` and call it something like `load_collection_from_yaml` or similar, it seems much easier to understand than having it in a separate module.

Reply:
Done!